src/System.Text.Json/src/System/Text/Json/Serialization/JsonNamingPolicy.cs

```diff
@@ -19,6 +19,11 @@ public abstract class JsonNamingPolicy
         /// </summary>
         public static JsonNamingPolicy CamelCase { get; } = new JsonCamelCaseNamingPolicy();
 
+        /// <summary>
+        /// Returns the naming policy for snake-casing.
```

Comment:
nit: Follow the content from the docs:
https://github.com/dotnet/dotnet-api-docs/blob/4f9dbb8474e1972f342b252a8d207e1155f41479/xml/System.Text.Json/JsonNamingPolicy.xml#L54-L55
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonnamingpolicy.camelcase?view=netcore-3.0

So:
```suggestion
        /// Gets the naming policy for snake-casing.
```

And add a value xml tag as well (which is equivalent to a "return" tag but for property getters).

Reply:
Should I do the same for camel case?

Reply:
> Should I do the same for camel case?

Sure, if you like. It wouldn't hurt doing a minor cleanup while you are in the similar area for this case. However, we currently don't back-port updates from the docs to the source since docs is the source of truth and we'd continuously get out-of-sync.

Reply:
Do you mean that documentation is written here first, than moved to the docs repo and never updated in the code? Just trying to understand the flow.

Reply:
> Do you mean that documentation is written here first, than moved to the docs repo and never updated in the code? Just trying to understand the flow.

Yes. The docs repo is the source of truth and contains "live" changes, along with review done by the doc writers during the initial move for correctness in terms of format/consistency/following guidelines. We don't currently have a great way to bring those changes back to code.